### ISSUE TITLE
Desktop Client sync pausing

### ISSUE THREAD

**Issue by JKL7** [owncloud/client#3441]
The documentation states...

"Pause/Resume button: Pauses the current sync (or prevents the client from starting a new sync), or resumes the sync process."

Pause did not stop a current sync, so I take it it is the "or" situation and while a sync is in progress, pause will only stop a new sync from starting.

Can or will this be changed? It would be great to be able and stop a running sync.

Windows 7 Home
Desktop Client 1.8.3 Build 5250

Thanks

**Comment by guruz** [comment]
The pause should always abort a current sync.
Do you have a situation where this does not happen?

**Comment by JKL7** [comment]
Yes, as stated above. Pics were added to the server, my Desktop client began to DL them, I had the bandwidth still set at full, so I quickly used the system tray icon to access the panel and asked it to pause the sync. The pics kept coming (a pop up was shown for each file DL'ed).

I quit the program to continue with another task. When I re-started the client there was no sync taking place so either all pics had already been DL'ed or they continued to DL even after I quit the client.

I since removed the existing pic folders as I do not care to sync the older photos.

Windows 7 Home
Desktop Client 1.8.3 Build 5250
